.github/workflows/ci.yml

```diff
@@ -8,34 +8,6 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
-
-    steps:
-    - uses: actions/checkout@v2
-
-    - name: Setup Java JDK
-      uses: actions/setup-java@v1.4.3
-      with:
-        java-version: 11
-
-    - name: Build with Maven
-      run: 'mvn clean verify --batch-mode 
-        -Daws.access.key=${{ secrets.AWS_ACCESS_KEY }} 
-        -Daws.secret.key=${{ secrets.AWS_SECRET_KEY }}
-        -Divy.engine.version=9.3.0
-        -Divy.engine.download.url=https://dev.axonivy.com/permalink/nightly/axonivy-engine.zip'
-
-    - name: Publish Unit Test Results
-      uses: EnricoMi/publish-unit-test-result-action@v1
-      if: always()
-      with:
-        files: |
-          */target/*-reports/*.xml
-          !*/target/*-reports/failsafe-summary.xml
```

Comment:
maybe we have to re-think of the Test pattern in the template (re-usable workflow).
I'd slightly prefer this solution here, rather than working with a TEST prefix as in the template.

Reply:
The problem with our template is, that we enforce a File pattern, which may be default ... but users can change it. Just think of the default test patterns supported by failsafe ... which can just be 'IT' 
Then this pattern here is better ... and it does no harm that used other patterns which is fine and supported by maven